hivemind_bus_client/client.py

```
@@ -87,7 +100,9 @@
                  compress: bool = True,
                  binarize: bool = True,
                  identity: NodeIdentity = None,
-                 internal_bus: Optional[OVOSBusClient] = None):
+                 internal_bus: Optional[OVOSBusClient] = None,
+                 bin_callbacks: BinaryDataCallbacks = BinaryDataCallbacks()):
```

Ruff

Do not perform function call `BinaryDataCallbacks` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)